WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleItemListEmptyView.swift

```diff
@@ -0,0 +1,58 @@
+import SwiftUI
+
+struct PointOfSaleItemListEmptyView: View {
+    var body: some View {
+        VStack(alignment: .center, spacing: Constants.headerSpacing) {
+            Text(Localization.productTitle)
```

Comment:
Same as with FloatingButton, it lacks respect for side margins.

Reply:
Yeah! I'll address this one as smaller PR separately, mainly because currently the main rendering is done inside the `headerTextView`, declared in the ItemList, and I want to extract this into a separate component we can reuse across all screens. It also seems to change colour in cases like "empty list of products", so these can be tackled together 👍 

Logged here: https://github.com/woocommerce/woocommerce-ios/issues/13559